src/styles/blocks/section2.scss

```diff
@@ -0,0 +1,67 @@
+.section_2 {
+  margin-top: 79px;
+  text-align: justify;
+  width: 166px;
+  height: 81px;
```

Comment:
Avoid setting explicit width and height on block elements; it's better to control the dimensions through padding and content. This ensures better responsiveness and maintainability.